Userland/Applications/Calculator/CalculatorWidget.h

```diff
@@ -15,9 +15,13 @@
 #include <LibGUI/Action.h>
 #include <LibGUI/Widget.h>
 
+namespace Calculator {
+
 class CalculatorWidget final : public GUI::Widget {
     C_OBJECT(CalculatorWidget)
 public:
+    static ErrorOr<NonnullRefPtr<CalculatorWidget>> create();
```

Comment:
We now have `create` and `try_create` here, both of which are fallible and one of which shouldn't be called from the outside.